### ISSUE TITLE
Option to customize the color of the arrow

### ISSUE THREAD

**Issue by tjhorner** [tjhorner/WhereIsForward#10]
Some users may not be able to see the default gray color of the arrow (or want to just set it to something else), so it would be nice to have a color picker in the settings.

Would also be pretty easy to add a "use chaperone bounds color" option.

This depends on #9 being completed.
